discovery-provider/tests/conftest.py

```diff
@@ -74,7 +75,7 @@ def contracts(app):  # pylint: disable=redefined-outer-name
     web3endpoint = "http://{}:{}".format(
         app.config["web3"]["host"], app.config["web3"]["port"]
     )
-    web3 = Web3(HTTPProvider(web3endpoint))
+    web3 = Web3(MultiProvider(web3endpoint))
```

Comment:
since we don't use `MultiProvider` for dataWeb3 this makes tests unrepresentative of actual state
its ok to leave it since it is our only unit test coverage, but lets comment explicitly that we are putting this here despite diverging from app state in order to get some coverage

Reply:
I think it should be representative of our actual flow for accuracy; this was mostly just me not  noticing and forgetting to change it back. I've flipped it back now.